crates/zksync_error/src/lib.rs

```diff
@@ -0,0 +1,325 @@
+//!
+//! AUTOGENERATED BASED ON A SET OF JSON FILES, DO NOT EDIT MANUALLY
+//!
+
+#![allow(unused)]
+pub mod documentation;
+pub(crate) mod error;
```

Comment:
```suggestion
pub mod error;
```

Access to types `CustomErrorMessage` and `NamedError`.

Reply:
Resolved by adding `pub use` for these types.